geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/CreateAlterDestroyRegionCommandsDUnitTest.java

```diff
@@ -1091,6 +1086,126 @@ public void testDestroyRegionWithSharedConfig() {
     });
   }
 
+
+  final String PR_STRING = " package com.cadrdunit;"
```

Comment:
@kirklund Thanks for the details review. I did not understand this comment. Could you please help me to understand this. I followed other examples from deploy jar tests